.github/workflows/ci.yml

```diff
@@ -18,3 +18,42 @@ jobs:
           cache: npm
       - run: npm ci
       - run: npm run test
+
+      - run: git fetch --unshallow
+        if: ${{ matrix.node-version == '16' }}
+
+      - name: SonarCloud Scan - Browser
+        if: ${{ matrix.node-version == '16' }}
+        uses: SonarSource/sonarcloud-github-action@v1.6
```

Comment:
It might be an idea to see if we can contribute tooling to this action to handle multiple projects at once, I'm not a huge fan of this style of one per package, as if we add additional packages it can create drift.